Dockerfile

```diff
@@ -1,12 +1,30 @@
+# Build JS app
+FROM node:current-alpine AS builder
+
+RUN apk --no-cache add yarn
+RUN mkdir -p /app/client
+WORKDIR /app/client
+
+COPY client/package*.json ./
```

Comment:
package-lock should be removed from version control + added to gitignore and `package.json` be the only file we need to copy